include/bluetooth/uuid.h

```diff
@@ -50,29 +50,29 @@ struct bt_uuid_128 {
 	u8_t val[16];
 };
 
-#define BT_UUID_INIT_16(value)		\
-{					\
-	.uuid.type = BT_UUID_TYPE_16,	\
-	.val = (value),			\
-}
+#define BT_UUID_INIT_16(value)		      \
+	{				      \
+		.uuid.type = BT_UUID_TYPE_16, \
+		.val = (value),		      \
+	}
 
-#define BT_UUID_INIT_32(value)		\
-{					\
-	.uuid.type = BT_UUID_TYPE_32,	\
-	.val = (value),			\
-}
+#define BT_UUID_INIT_32(value)		      \
+	{				      \
+		.uuid.type = BT_UUID_TYPE_32, \
+		.val = (value),		      \
+	}
 
-#define BT_UUID_INIT_128(value...)	\
-{					\
-	.uuid.type = BT_UUID_TYPE_128,	\
-	.val = { value },		\
-}
+#define BT_UUID_INIT_128(value ...)	       \
+	{				       \
+		.uuid.type = BT_UUID_TYPE_128, \
+		.val = { value },	       \
+	}
```

Comment:
These white-space changes don't seem to be related to the fix? Please put them in a separate patch. We'd probably want to merge this for 2.0, and the less changes there are the better.